### DISCUSSION TITLE
Can't publish migration because of QueryException

### DISCUSSION THREAD

**Discussion by genesiscz** [spatie/laravel-translation-loader#123] [Q&A]
So there was an issue https://github.com/spatie/laravel-translation-loader/issues/80 which shown its not possible to install the package without manually creating a table. There even was a pull request which was closed. Is there a fix for this?

**Comment by freekmurze** [comment]
I haven't encountered this problem myself. It's also been a while since I used this package myself.

Like stated in #80, I'd accept a PR that fixes this.

**Comment by genesiscz** [comment]
Is there a way to disable autodiscovery or at least publish the assets while autodiscovery process? Not too experienced in this field of Laravel so not sure

**Comment by mnazmi23** [comment]
I am having this issue too. Had to remove the package first then proceed with my setup. I dont want to create the table manually though.

**Comment by spoyntersmith** [comment]
Are any of you using Laravel Jetstream in you application? @mnazmi23 @genesiscz

**Reply by spoyntersmith** [reply]
The reason is that the JetstreamServiceProvider has translations which if used before the table is created breaks

Might be helpful to know about / add a check to see if the language table exists before attempting to load it?
